distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -198,8 +198,8 @@ public void tearDown() throws IOException {
     @Test
     public void testCreateStorage() throws IOException {
         String fileName = testName.getMethodName() + ".jpg";
-        File tmpFile = createTemporaryFile(fileName, IOUtils.toInputStream(getFileContent(
-                SAMPLE_IMAGE)));
+        File tmpFile = createTemporaryFile(fileName,
+                IOUtils.toInputStream(getFileContent(SAMPLE_IMAGE)));
```

Comment:
❓ There is a lot of "reformatting noise" in this diff. Could we just have the actual changes in this diff and keep the reformatting changes in a separate commit/PR?